go/cmd/dolt/cli/arg_parser_helpers.go

```diff
@@ -290,6 +290,17 @@ func CreateLogArgParser() *argparser.ArgParser {
 	ap.SupportsString(DecorateFlag, "", "decorate_fmt", "Shows refs next to commits. Valid options are short, full, no, and auto")
 	ap.SupportsFlag(OneLineFlag, "", "Shows logs in a compact format.")
 	ap.SupportsStringList(NotFlag, "", "revision", "Excludes commits from revision.")
+	return ap
+}
+
+func CreateLogTableArgParser() *argparser.ArgParser {
+	ap := argparser.NewArgParserWithVariableArgs("log")
+	ap.SupportsInt(NumberFlag, "n", "num_commits", "Limit the number of commits to output.")
+	ap.SupportsInt(MinParentsFlag, "", "parent_count", "The minimum number of parents a commit must have to be included in the log.")
+	ap.SupportsFlag(MergesFlag, "", "Equivalent to min-parents == 2, this will limit the log to commits with 2 or more parents.")
+	ap.SupportsFlag(ParentsFlag, "", "Shows all parents of each commit in the log.")
+	ap.SupportsString(DecorateFlag, "", "decorate_fmt", "Shows refs next to commits. Valid options are short, full, no, and auto")
+	ap.SupportsStringList(NotFlag, "", "revision", "Excludes commits from revision.")
```

Comment:
To reduce duplication, I would do something like this:

ap := cli.CreateLogArgParser()
ap.SupportsStringList(TableFlag...